### PR TITLE
Issue #3 update openapi setup

### PR DESCRIPTION
**Pull request by Goldziher**
This PR resolves issue #3. See the update to. the readme example to see how its supposed to be used now.

### CODE REVIEW

**Review by cofin — APPROVED** [review]
Can you take a look at the changes I added in?  I could not get the test cases for `exp` and `iat` fields in `Token`  to pass without modification.  They were failing validation due to local time to UTC comparison issues.

File: pyproject.toml [hunk 1]
```diff
@@ -44,6 +44,7 @@ requests = "*"
 pre-commit = "*"
 hypothesis = "*"
 mkdocs-gen-files = "*"
+types-python-jose = "*"
```
Comment (Goldziher) [inline pyproject.toml:47]: you can install this as part of the `.pre-commit-config.yaml` by adding it to the mypy `additional_dependencies` - unless its required here.

File: starlite_jwt/jwt_auth.py [hunk 2]
```diff
@@ -173,7 +173,7 @@ def create_token(
         """
         token = Token(
             sub=identifier,
-            exp=(datetime.utcnow() + (token_expiration or self.default_token_expiration)),
+            exp=(datetime.now(timezone.utc) + (token_expiration or self.default_token_expiration)),
```
Comment (Goldziher) [inline starlite_jwt/jwt_auth.py:176]: you must ensure that timezone is backwards compatible with py3.8 and py3.7. Im not sure it is.

**Review by Goldziher — COMMENTED** [review]
looks good to me - merge when ready

**Pull request merged**